geetools/ee_dictionary.py

```diff
@@ -80,3 +82,42 @@ def getMany(self, list: list | ee.List) -> ee.List:
                 d.geetools.getMany(["foo", "bar"]).getInfo()
         """
         return ee.List(list).map(lambda key: self._obj.get(key))
+
+    def toTable(self, valueType: ee.List | ee.Dictionary | Any = Any) -> ee.FeatureCollection:
+        """Convert a ee.Dictionary to a ee.FeatureCollection with no geometries (table).
+
+        The keys will always be stored in `system:index` column.
+
+        Parameters:
+            valueType: this will define how to process the values. In case of
+            ee.List the values will be stored in columns named `value_<position>`.
+            In case of a ee.Dictionary, column names will be created from the keys.
+            For any other type, it will return a table with one feature with
+            one column per key.
+
+        Returns:
+            a ee.FeatureCollection in which the keys of the ee.Dictionary are
+            in the `system:index` and the values are in new columns.
+        """
+
+        def features_from_dict(key, value) -> ee.Feature:
+            key = ee.String(key)
+            feat = ee.Feature(None, {"system:index": key})
+            return feat.set(ee.Dictionary(value))
+
+        def features_from_list(key, value) -> ee.Feature:
+            key = ee.String(key)
+            feat = ee.Feature(None, {"system:index": key})
+            value = ee.List(value)
+            keys = ee.List.sequence(1, value.size())
+            keys = keys.map(lambda k: ee.String("value_").cat(ee.Number(k).toInt()))
+            properties = ee.Dictionary.fromLists(keys, value)
+            return feat.set(properties)
```

Comment:
Alternative building blocks: 
```suggestion
            index = {"system:index" : ee.String(key)}
            values = ee.List(value)
            columns = ee.List.sequence(1, value.size())
            columns = columns.map(lambda k: ee.String("value_").cat(ee.Number(k).toInt()))
            props = ee.Dictionary.fromLists(columns, values).combine(index)
            return ee.Feature(None, props)
```